### PR TITLE
Setup shared component for rendering teams or lenders sections. This …

### PR DESCRIPTION
**Pull request by mcstover**
…includes the placeholder content and initial data fetch.

image

### CODE REVIEW

**Review by BoulderBrains — APPROVED** [review]
nice

File: src/components/BorrowerProfile/LendersAndTeams.vue [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+					<div>
+						<span v-if="item.name">{{ item.name }}</span><br>
+						<span v-if="item.lenderPage && item.lenderPage.whereabouts">
+							{{ item.lenderPage.whereabouts }}
+						</span>
+					</div>
+				</div>
+			</div>
+
+			<div
```
Comment (ryan-ludwig) [inline src/components/BorrowerProfile/LendersAndTeams.vue:58]: Please use a `button` tag to set the correct role and allow keyboard focus

Comment (mcstover) [reply src/components/BorrowerProfile/LendersAndTeams.vue:58]: Will do in the follow up ticket for building out the lightbox.

File: src/components/BorrowerProfile/LendersAndTeams.vue [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+					{{ poweredByText }}
+				</span>
+			</div>
+
+			<div v-if="loading" class="tw-grid tw-grid-cols-2 tw-gap-2">
+				<!-- Loading placeholder for team or lender item elements -->
+				<div v-for="i in 5" :key="i" class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-mb-3">
+					<kv-loading-placeholder class="tw-rounded md:tw-hidden tw-mb-1.5" :style="{height: '128px'}" />
+					<kv-loading-placeholder
+						class="tw-rounded md:tw-block tw-hidden tw-mr-2" :style="{width: '88px', height: '88px'}"
```
Comment (ryan-ludwig) [inline src/components/BorrowerProfile/LendersAndTeams.vue:30]: Adding `flex-shrink-0` will prevent these squares from getting squished
image

File: src/components/BorrowerProfile/LendersAndTeams.vue [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+			<div v-if="!loading" class="section-items tw-mb-3 tw-grid tw-grid-cols-2 tw-gap-2">
+				<!-- Replace with Generic component for showing Either Team or Lender image and information -->
+				<div
+					v-for="(item, index) in truncatedItemList" :key="index"
+					class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-mb-3"
+				>
+					<kv-loading-placeholder class="tw-rounded md:tw-hidden tw-mb-1.5" :style="{height: '128px'}" />
+					<kv-loading-placeholder
+						class="tw-rounded md:tw-block tw-hidden tw-mr-2" :style="{width: '88px', height: '88px'}"
```
Comment (ryan-ludwig) [inline src/components/BorrowerProfile/LendersAndTeams.vue:47]: same flex-shrink-0 issue as above

**Review by ryan-ludwig — APPROVED** [review]
Nice, digging the lazy load

File: src/components/BorrowerProfile/LendersAndTeams.vue [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+				/>
+				<span v-else>
+					{{ poweredByText }}
+				</span>
+			</div>
+
+			<div v-if="loading" class="tw-grid tw-grid-cols-2 tw-gap-2">
+				<!-- Loading placeholder for team or lender item elements -->
+				<div v-for="i in 5" :key="i" class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-mb-3">
+					<kv-loading-placeholder class="tw-rounded md:tw-hidden tw-mb-1.5" :style="{height: '128px'}" />
```
Comment (ryan-ludwig) [inline src/components/BorrowerProfile/LendersAndTeams.vue:28]: Minor, but I'm guessing we want to move this into the kv-ui-elements repo soon?

File: src/components/BorrowerProfile/LendersAndTeams.vue (hunk 2 above)
Comment (mcstover) [reply src/components/BorrowerProfile/LendersAndTeams.vue:30]: Indeed, these are just placeholders until we drop in the real component.

File: src/components/BorrowerProfile/LendersAndTeams.vue (hunk 4 above)
Comment (mcstover) [reply src/components/BorrowerProfile/LendersAndTeams.vue:28]: Yea we were kinda waiting on design to proved an update but I really think we could just port as is and tweak later...

File: src/components/BorrowerProfile/LendersAndTeams.vue (hunk 3 above)
Comment (mcstover) [reply src/components/BorrowerProfile/LendersAndTeams.vue:47]: @BoulderBrains the classes and techniques on this section may be useful for your upcoming component work. The tips @ryan-ludwig has stated around the `flex-shrink-0` setting will take it even farther toward a usable layout!

**Pull request merged**